README.md

```diff
@@ -13,7 +13,9 @@ with OpenCensus, they have to modify their code to register an
 exporter and redeploy. Asking our users recompile and redeploy is
```

Comment:
Also another case is if somebody wants to distribute a binary instrumented with OC which export should they use, with the agent they can distribute the binary with the agent exporter and the user will run the agent with the exporter of choice.